src/accounts/molecules/ShowQr.tsx

```diff
@@ -12,14 +13,14 @@ export default function QROrLoading({
     <>
       {!isMobile() && (
         <>
-          <TP>Scan the QR code with the Tonomy ID app</TP>
+          <TP>Scan the QR code with the {settings.config.ecosystemName} app</TP>
```

Comment:
should be the App Name, not ecosystem name
Tonomy ID | Pangea Passport Testnet--> appName
Tonomy | Tonomy Staging | Pangea --> ecosystemName

Please check all locations.